src/emu/debugger/src/elements/imgui_debugger_prefs.cpp

```diff
@@ -54,7 +54,7 @@ namespace eka2l1 {
 
         const std::string background_str = common::get_localised_string(localised_strings, "pref_personalize_background_string");
 
-        ImGui::Text(background_str.c_str());
+        ImGui::Text("%s",background_str.c_str());
```

Comment:
Can you ask space after the comma? It's a codestyle which you can see around here.